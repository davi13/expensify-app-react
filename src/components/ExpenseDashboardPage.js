import React from 'react';
//import ExpensesSmmary from '../components/ExpensesSummary'
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters />
        <ExpenseList />
        
    </div>
);

export default ExpenseDashboardPage;
