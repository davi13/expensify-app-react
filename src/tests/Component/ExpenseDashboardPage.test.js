import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('Should render ExpenseDashboard', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot()
})