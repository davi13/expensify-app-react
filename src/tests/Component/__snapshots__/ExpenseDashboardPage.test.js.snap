// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashboard 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
